src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -33,54 +89,28 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton id={txDetails.txId} />
         </div>
       )}
-      <div className="tx-hash">
-        <Text size="xl" strong as="span">
-          Transaction hash:{' '}
-        </Text>
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
+      {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" />}
+      {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
+      <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
+      <br></br>
```

Comment:
I would only use `<br>` for breaking a line of text. This can be done with CSS.

Reply:
Can you suggest a snippet here? I see we are using `<br />` in more places that's why I didn't hesitate in also using it here

Reply:
Add a wrapper with a margin-bottom?